src/services/syncFeedPerUser.js

```diff
@@ -134,20 +174,20 @@ const resetAndSyncFeed = async (req, res) => {
     const followinfFeed = await listFeedFollowing(feed, userId);
     console.log(`Start reset feed ${feed}`);
     await Promise.all(
-      followinfFeed.map(async (feed) => {
-        const targetFeed = feed.target_id.split(':');
+      followinfFeed.map(async (fd) => {
+        const targetFeed = fd.target_id.split(':');
         console.log(targetFeed);
         const result = await clinetFeed.unfollow(targetFeed[0], targetFeed[1]);
         console.log(result);
       })
     );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Promise.all` function is used here to execute multiple promises concurrently, which can improve performance. However, if any of the promises fail, `Promise.all` will immediately reject with the reason of the first promise that rejected, discarding all the other promises whether they fulfilled or rejected. This could potentially lead to unhandled promise rejections. Consider using `Promise.allSettled` instead, which waits for all promises to settle, regardless of whether they fulfilled or rejected.
```diff
  await Promise.all(
-    followinfFeed.map(async (fd) => {
+    followinfFeed.map((fd) => {
      const targetFeed = fd.target_id.split(':');
      console.log(targetFeed);
-     const result = await clinetFeed.unfollow(targetFeed[0], targetFeed[1]);
+     return clinetFeed.unfollow(targetFeed[0], targetFeed[1])
-     console.log(result);
-   })
+   })).then(results => results.forEach(result => console.log(result)));
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->